components/supervisor/frontend/src/index.ts

```diff
@@ -30,6 +31,15 @@ window.addEventListener('error', (event) => {
             labels['error'] = 'LoadError';
         }
     }
+    if (event.error) {
+        IDEMetricsServiceClient.reportError(event.error).catch(() => {});
+    } else if (labels['error'] == 'LoadError') {
+        let error = new Error("LoadError")
```

Comment:
Each event should end up with error, otherwise we can miss something like script was loaded but failed on execution.

Reply:
How do you know it there is nothing? Why when it is called? At least we should report that it is happening. It sounds more like we don't understand args. 

As far as I understand it can be `message: string, filename: string, line: number, column?: number, e?: any`?  so we need to check if first argument is string, then we should compose error structure from such, if everything else fails then at least report `UnknownErrror`

Reply:
We should not swallow anything.